pages/settings.tsx

```diff
@@ -38,7 +38,7 @@ const Settings = () => {
               <h1 className="text-xs-center">{title}</h1>
               <SettingsForm />
               <hr />
-              <button className="btn btn-outline-danger" onClick={handleLogout}>
+              <button data-cy="logout-btn-settings" className="btn btn-outline-danger" onClick={handleLogout}>
```

Comment:
There is no major issue in the code. Data-cy attribute is added as required.

Reply:
changed